.makim.yaml

```
@@ -0,0 +1,83 @@
+version: 1.0.0
+groups:
+  - name: tests
+    targets:
+      linter:
+        help: Use this target to review linter
+        run: |
+          pre-commit run --all-files --verbose
+          poetry run darglint --verbosity 2 epigraphhub_py tests
+
+      unittest:
+        help: Use this target to test code
+        args:
+          extra:
+            help: Extra argument for pytest
+            type: string
+            default: ''
+        run: |
+          poetry run pytest --cov-report=html -vv --cov tests/ {{ args.extra }}
+          poetry run coverage-badge -o assets/images/coverage.svg -f
+
+      check-safety:
+        help: Review the checking safety of the code
+        run: |
+          poetry check
+          poetry run safety check --full-report
+          poetry run bandit -ll --recursive epigraphhub_py tests
+
+  - name: config
+    targets:
+      create-file:
+        help: Configuration file
+        run: |
+          bash scripts/dev/create-config-file.sh
+
+  - name: containers
```

```suggestion
  containers:
```

change needed by makim 1.6.4